### DISCUSSION TITLE
Export to Excel

### DISCUSSION THREAD

**Discussion by setin1219** [vikramlearning/blazorbootstrap#634] [Q&A] [answered]
Hi, 

Are grids supported with "export to excel/csv" functionality?
Let me know if this is achievable out of the box with blazor bootstrap.

**Comment by gvreddy04** [comment, marked as answer]
@setin1219 Export option is not yet supported. You can track a similar issue here https://github.com/vikramlearning/blazorbootstrap/issues/328.

**Reply by setin1219** [reply]
@gvreddy04  thank you for the update, yes this will be a nice feature to include.

**Reply by heather-timmins** [reply]
I second this - Would be a brilliant feature for the grid
